packages/components/src/FormatFile/FormatFile.css

```diff
@@ -51,12 +51,61 @@
   margin: 0 auto;
 }
 
-.progress {
+.fileContentWrapper {
+  display: flex;
+  width: inherit;
+  height: inherit;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.thumbailIconWrapper {
+  align-self: center;
+}
+
+.thumbnailFilename {
   display: flex;
   position: absolute;
-  top: 0;
+  bottom: var(--space-small);
+  left: var(--space-small);
+}
+
+.thumbnailFilenameSmall {
```

Comment:
This text size needs to be 10px instead of 12. 14 in the large thumbnail, 10 in the small.

Reply:
We had initially only found in FontSizes.css that it went to `smaller` but I've done further digging to get the font to 10px using `font-size: calc(var(--base-unit) * 0.625);` 👍